MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -15175,19 +15174,19 @@ void ApplicationWindow::windowProperties() {
   s += tr("Label") + ": " + static_cast<MdiSubWindow *>(w)->windowLabel() +
        "\n\n";
 
-  if (w->isA("Matrix")) {
+  if (QString(w->metaObject()->className()) == "Matrix") {
     mbox->setIconPixmap(getQPixmap("matrix_xpm"));
     s += tr("Type") + ": " + tr("Matrix") + "\n\n";
-  } else if (w->inherits("Table")) {
+  } else if (QString(w->metaObject()->className()) == "Table") {
```

Comment:
This one was `inherits` (rather than `isA` like the other lines) which is still ok in Qt5, I think. Would the changed version match something that derived from a Table e.g. MantidTable?